packages/container/libraries/strongly-typed/src/inject.spec.ts

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable jest/expect-expect */
+import { describe, it } from '@jest/globals';
+
+import 'reflect-metadata';
+
+import { inject, injectable } from 'inversify';
+
+import type { TypedInject } from './inject';
+
+describe('@inject', () => {
+  @injectable()
+  class Foo {
+    public foo: string = '';
+  }
+
+  @injectable()
+  class Bar {
+    public bar: string = '';
+  }
+
+  interface BindingMap {
+    foo: Foo;
+    bar: Bar;
+    asyncNumber: Promise<number>;
```

Comment:
We should add a test involving this field

Reply:
Ha added this and completely forgot to test it 🤦🏼